nodes/vector/turbulence.py

```diff
@@ -121,22 +132,24 @@ def process(self):
             for vertex in verts[0]:
                 # set the offset origin for random seed
                 if rseed == 0:
-                    #we set offset value to 0.0
-                    offset = [0.0,0.0,0.0]
+                    # we set offset value to 0.0
+                    offset = [0.0, 0.0, 0.0]
                 else:
                     # randomise origin
-                    noise.seed_set( rseed )
+                    noise.seed_set(rseed)
                     offset = noise.random_unit_vector() * 10
 
-                new_vertex = [ x+y for x, y in zip(vertex, offset)]
+                new_vertex = [x + y for x, y in zip(vertex, offset)]
                 coords.append(new_vertex)
 
-            out.append([turbulence_function(v, m_octaves, m_hard, _noise_type, m_amp, m_freq) for v in coords])
+            out.append([turbulence_function(v, m_octaves, m_hard,
+                                            _noise_type, m_amp, m_freq) for v in coords])
```

Comment:
unless you planned to make the params different per inputted set of coords too, (_fully vectorized_).